src/pages/developing/web-components-tutorial/step-4.mdx

```diff
@@ -0,0 +1,365 @@
+directory and not continuing from the previous step):
+
+```bash
+pnpm i
+```
+
+Then, start the app:
+
+```bash
+pnpm dev
+```
+
+You should see something similar to where the
+[previous step](/developing/web-components-tutorial/step-1#fork-clone-and-branch/step-3)
+left off.
+
+## Review design
+
+Here's what we're building – an informational section that has a heading and
+three subheadings. Each subheading has accompanying copy and a pictogram. We'll
+assume that this informational section is used elsewhere on the site, meaning
+it's a great opportunity to build it as a reusable component. As for naming,
+we'll call it an `InfoSection` with three `InfoCard`s as children.
+
+![Info section layout](../shared/step-4/images/info-layout.png)
+
+<Caption>Info section layout</Caption>
+
+## The info section
+
+Inside `page-landing__r3` in `index.html` we will make a couple of small
+changes. Add the class `info-section` to the `cds--subgrid`
+
+```html path=index.html
+<div class="info-section cds--subgrid cds--subgrid--full-wide"></div>
+```
+
+and replace `The principles` and surrounding column with which changes the
+column settings and provides a class for us to style the section heading.
+
+```html path=index.html
+<div
+  class="info-section__column cds--sm:col-span-4 cds--md:col-span-8 cds--lg:col-span-16 cds--xlg:col-span-4 cds--css-grid-column">
+  <h3 class="info-section__heading">The Principles</h3>
+</div>
+```
+
+Style the `info-section__heading` with:
+
+```scss path=style.scss
+.info-section__heading {
+  @include type-style('heading-01');
+
+  padding-block-end: $spacing-08;
+}
+```
+
+## The info card
+
+At the bottom of `index.html` between the closing body and html tags add the
+info card template.
+
+### The template
+
+```html path=index.html
+<template id="template--info-card">
+  <div
+    class="info-card cds--sm:col-span-4 cds--md:col-span-8 cds--lg:col-span-5 cds--xlg:col-span-4 cds--css-grid-column">
+    <div class="info-card__uppder">
+      <h4 class="info-card__heading">
+        Carbon is
+        <strong class="info-card__heading--strong">thing goes here</strong>
+      </h4>
+      <p class="info-card__body">Body goes here</p>
+    </div>
+    <div class="info-card__pictogram"></div>
+  </div>
+</template>
+```
+
+Before we make use of the template we need to remove the column settings from
+the remaining three elements `page-landing__title` and replace with `info-card
+to leave:
+
+```html index.html
+<div class="info-card">Carbon is open</div>
+<div class="info-card">Carbon is modular</div>
+<div class="info-card">Carbon is consistent</div>
+```
+
+### The script
+
+As the script in this file only affects our landing page, first lets create
+`landing.js`. Kicking things off by moving the breadcrumb and tabs import from
+`main.js`.
+
+```javascript path=landing.js
+import '@carbon/web-components/es/components/breadcrumb/index';
+import '@carbon/web-components/es/components/tabs/index';
+```
```

Comment:
are these imports necessary since they are already in `main.js`

Reply:
The intention is to move all code specific to the landing page, to landing.js